Tests/FeaturevisorSDKTests/InstanceTests.swift

```diff
@@ -5,6 +5,89 @@ import XCTest
 
 class FeaturevisorInstanceTests: XCTestCase {
 
+    func testEncodeEvaluationReturnsValidDatafileContent() throws {
+
+        //GIVEN
+        let traffic = Traffic(key: "key", segments: .plain("segment"), percentage: 13, allocation: [])
+        let overrideFeature = OverrideFeature(enabled: false, variation: "variation", variables: [:])
+        let variation = Variation(description: "description", value: "value", weight: 3, variables: [])
+        let variableSchema = VariableSchema(key: "key", type: .object, defaultValue: .object(["": .boolean(false)]))
+
+        let evaluation = Evaluation(
+            featureKey: "feature123",
+            reason: .allocated,
+            bucketValue: 42,
+            ruleKey: "rule456",
+            enabled: true,
+            traffic: traffic,
+            sticky: overrideFeature,
+            initial: overrideFeature,
+            variation: variation,
+            variationValue: "value",
+            variableKey: "color",
+            variableValue: .string(""),
+            variableSchema: variableSchema
+        )
+
+        let encoder = JSONEncoder()
+        let jsonData = try encoder.encode(evaluation)
+
+        guard let decodedDictionary = try JSONSerialization.jsonObject(with: jsonData, options: []) as? [String: Any] else {
```

Comment:
This is your "when". Under "when" you put the code which you actually want to assert/test. So don't put there mock expected data (btw. it should be under "GIVEN" section) but this part